### DISCUSSION TITLE
Trying to import POIs into OsmAnd

### DISCUSSION THREAD

**Discussion by Panissal** [osmandapp/OsmAnd#12838] [Maps Related]
I have over the years created, on my TomTom GO Live 825, 4 groups of ov2 waypoints (850 in all) of places I want to visit or have visited. I have exported them to my computer, then converted them to kml format for importing into Magic Earth navigator. That works fine and all waypoints appear in the original 4 POI folders. Now I am trying to do the same thing with OsmAnd. So I have converted the ov2 folders to gpx folders and tried to import them, only to be faced with the message "Import tracks Cannot import specified GPX file". Does anyone know if I'm just being dense, because I'm trying to do the impossible, or could it be that until I download all the relevant maps (France, England, Scotland, Wales, Germany, Belgium, Holland, Switzerland and Italy) the app won't accept the files because most of them won't have a 'home'. I have tried changing the name of the folder to favourites.gpx (to match the folder name that the favourites that I have created in the app are saved to); I have tried creating a new My Places folder to match the name of the folder that my waypoints are stored in. But I still get the same error message.
Anyone got any ideas? Thank you
Panissal

**Comment by pebogufi** [comment]
Is it possible to upload one of your gpx files in ZIPped format here, or does it hurt some privacy ?
Checking the file could help to help you.

**Comment by Panissal** [comment]
No privacy implications that I can think of, except to know that I like Romanesque churches! The zipped kml file worked fine on Magic Earth but the equivalent gpx file always fails on OsmAnd. I did the conversion on POI Editor but I’m puzzled that, unzipped the kml file was 59 KB whereas the gpx file was 25 KB. Does that make sense? And zipped the kml is smaller than the gpx.
Thank you for your help

**Reply by pebogufi** [reply]
I personally never use kml, so i can't ckeck that. 
The technical xml structure of *your* gpx file should be checked (compared to favourites.gpx) to understand what's going wrong.

**Comment by Panissal** [comment]
Hi
I’ve only got one entry in my ‘official’ favourites file. I was rather h hoping I wouldn’t have to enter all 850 by hand. Would one be enough to check the xml structure with. 
I guess it’s pretty obvious that’s rather beyond my comfort zone,I’m afraid
